guides/common/modules/con_using-pxe-to-provision-hosts.adoc

```diff
@@ -43,22 +43,27 @@ In {Project} provisioning, the PXE loader option defines the DHCP `filename` opt
 * For BIOS systems, select the *PXELinux BIOS* option to enable a provisioned host to download the `pxelinux.0` file over TFTP.
 * For UEFI systems, select the *Grub2 UEFI* option to enable a TFTP client to download `grubx64.efi` file, or select the *Grub2 UEFI HTTP* option to enable an UEFI HTTP client to download `grubx64.efi` with the HTTP Boot feature.
 
-ifndef::satellite[]
 {ProjectName} supports UEFI Secure Boot.
 SecureBoot PXE loaders enable a client to download the `shim.efi` bootstrap boot loader that then loads the signed `grubx64.efi`.
 Use the *Grub2 UEFI SecureBoot* PXE loader for PXE-boot provisioning or *Grub2 UEFI HTTPS SecureBoot* for HTTP-boot provisioning.
 
+ifdef::satellite[]
+By default, you can provision the RHEL version of your {ProjectServer} on Secure Boot enabled hosts.
```

Comment:
Is it intentional that `RHEL` is not enclosed in curly brackets here?

Reply:
Yes, it is. I'm using the abbreviation for better readability.

Reply:
It's okay to use abbreviations. Typically, we expand abbreviations upon first use in a section or chapter to introduce its meaning (`Red Hat Enterprise Linux (RHEL)`) and then it's prefectly fine to keep using the abbreviation only (`RHEL`). Not sure what the current practice is with RHEL itself, that particular abbreviation is so common that it might not need expanding on each first use :shrug: